jekyll-kuma-plugins/lib/jekyll/kuma-plugins/liquid/tags/policyschema.rb

```diff
@@ -0,0 +1,47 @@
+module Jekyll
+  module KumaPlugins
+    module Liquid
+      module Tags
+        class PolicySchema < ::Liquid::Tag
+          def initialize(tag_name, markup, options)
+            super
+      
+            @markup = markup.strip
+          end
+      
+          def render(context)
+            release = context.registers[:page]['release']
+      
+            <<~TIP
+              <div id='markdown_html'></div>
+      
+              <script type="text/javascript">
+              document.addEventListener("DOMContentLoaded", function(){
+                  const schema = fetch("/docs/#{release}/generated/#{@markup}.json")
```

Comment:
Should we make this path configurable in the tag? Feels like we may need to have something different for KM. (We can probably do this in a follow up though)

Reply:
ideally this would be configurable for the entire project.

Reply:
https://github.com/kumahq/kuma-website/issues/1325